packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: 0.9.2
\ No newline at end of file
+    version: 1.0.0
```

Comment:
The test failure may be caused by the change in dependency? Can you recover the change in `packages.yml` file?